Client/Frontend/Browser/BrowserViewController/BrowserViewController+WebViewDelegates.swift

```diff
@@ -695,6 +696,12 @@ extension BrowserViewController: WKNavigationDelegate {
                     }
                 }
             }
+
+            if !User.shared.searched, let url = webView.url,
+               url.isEcosia, url.path.hasPrefix("/search") {
```

Comment:
In the feedback PR to your Core PR I added a constant for search, so we don't type `"/search"` in many places.